cf-agent/cf-agent.c

```diff
@@ -84,6 +84,8 @@
 #include <string_lib.h>
 #include <cfnet.h>
 #include <repair.h>
+#include <sys/types.h>                  /* checking umask on writing setxid log */
```

Comment:
Please add a proper title to the commit message before the long description.

Reply:
Ack. :)